src/pages/api/v1/job-descriptions/[id].ts

```diff
@@ -0,0 +1,90 @@
+import prisma from '../../../../lib/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const getJobDescription = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    if (typeof req.query.id === 'string') {
+      const { id } = req.query;
+      const data = await prisma.jobDescription.findUnique({
+        where: { id: parseInt(id) },
+      });
+      if (data) {
+        res.status(200).json(data);
+      } else {
+        res
+          .status(404)
+          .json({ message: `Could not find Job Description with id ${id}` });
+      }
+    }
+  } catch (error) {
+    res.status(400).json({
+      message: `Invalid Input: 'id' was not a number`,
+    });
+  }
+  if (req.method === 'PATCH') {
+  } else if (req.method === 'DELETE') {
+  }
+};
+
+const updateHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    if (typeof req.query.id === 'string') {
+      const { id } = req.query;
+      const { rawText } = req.body;
+      const data = await prisma.jobDescription.update({
+        where: { id: parseInt(id) },
+        data: { rawText },
+      });
+      if (data) {
+        res.status(200).json(data);
+      }
```

Comment:
Need to consider what happens when data is falsy. We'll worry about this one later!